src/pages/events/EventShow.jsx

```diff
@@ -7,265 +7,321 @@ import "./Events.scss";
 import EventSignIn from "./EventSignIn";
 
 export default function EventShow() {
+  const [isLoading, setIsLoading] = useState(true);
   const { path, url } = useRouteMatch();
   const { event_id } = useParams();
-  const event = {
-    "id": event_id,
-    "name": `Event ${event_id}`,
-    "format": "NLHE",
-    "start_date": new Date("2020-02-01T19:00:00"),
-    "notes": "First testing",
-    "count": 3,
-    "state": 0
+  const [event, setEvent] = useState({
+    id: event_id,
+    name: "",
+    start_date: "",
+    format: "",
+    notes: "",
+    semester_id: "",
+    state: 0
+  });
+  const [participants, setParticipants] = useState([]);
+  const [filteredParticipants, setFilteredParticipants] = useState([]);
+  const [search, setSearch] = useState("");
+
+  const [error, setError] = useState("");

```

Comment:
Unless I'm mistaken you don't actually use this error anywhere besides setting the value. If that is the case just remove it

Reply:
Looking at it now you save the status, which is just a number. Either remove it or take the error message from the body instead